Sdk/Track/Swrve.m

```diff
@@ -1824,7 +1824,10 @@ - (void) queueDeviceProperties
     for (NSString* key in deviceProperties) {
         [formattedDeviceData appendFormat:@"  %24s: %@\n", [key UTF8String], [deviceProperties objectForKey:key]];
     }
-    DebugLog(@"Swrve config:\n%@", formattedDeviceData);
+    
+    if (!getenv("RUNNING_UNIT_TESTS")) {
```

Comment:
You mentioned that RUNNING_UNIT_TESTS was used someplace else in the system? Where?